django/core/management/commands/migrate.py

```diff
@@ -203,6 +203,8 @@ def handle(self, *args, **options):
             targets, plan=plan, state=pre_migrate_state.clone(), fake=fake,
             fake_initial=fake_initial,
         )
+        if post_migrate_state.is_delayed and 'apps' in post_migrate_state.__dict__:
```

Comment:
This pattern is repeated -- would it be appropriate as a method on `ProjectState` instead?